bin/BuildPackages.sh

```diff
@@ -1,5 +1,15 @@
 #!/usr/bin/env bash
 
+LOGDIR=log
+if [ ! -d "$LOGDIR" ]
+  then
+  mkdir -p $LOGDIR
+fi
+LOGFILE=buildpackages
+FAILPKGFILE=fail
+
+build_all_packages() {
+
 set -e
```

Comment:
Hmm, why is the `set -e` here -- why not have it active for the whole script? I.e. change the shebang to
```
#!/usr/bin/env bash -e
 `